test/integration/projects/controls/development.rb

```diff
@@ -0,0 +1,137 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dev_bu1_project_base = attribute('dev_bu1_project_base')
+dev_bu1_project_floating = attribute('dev_bu1_project_floating')
+dev_bu1_project_restricted_id = attribute('dev_bu1_project_restricted')
+dev_bu1_project_restricted_number = attribute('dev_bu1_project_restricted_number')
+dev_bu1_restricted_vpc_service_control_perimeter_name = attribute('dev_bu1_restricted_vpc_service_control_perimeter_name')
+dev_bu1_restricted_enabled_apis = attribute('dev_bu1_restricted_enabled_apis')
+access_context_manager_policy_id = attribute('access_context_manager_policy_id')
+
+dev_bu2_project_base = attribute('dev_bu2_project_base')
+dev_bu2_project_floating = attribute('dev_bu2_project_floating')
+dev_bu2_project_restricted_id = attribute('dev_bu2_project_restricted')
+dev_bu2_project_restricted_number = attribute('dev_bu2_project_restricted_number')
+dev_bu2_restricted_vpc_service_control_perimeter_name = attribute('dev_bu2_restricted_vpc_service_control_perimeter_name')
+dev_bu2_restricted_enabled_apis = attribute('dev_bu2_restricted_enabled_apis')
+
+control 'gcp-development' do
+  title 'gcp step 4-projects test development'
+
+  describe google_project(project: dev_bu1_project_floating) do
+    it { should exist }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: dev_bu1_project_restricted_id) do
+    it { should exist }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: dev_bu1_project_base) do
+    it { should exist }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: dev_bu2_project_floating) do
+    it { should exist }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: dev_bu2_project_restricted_id) do
```

Comment:
@amandakarina I think a good test would be if the projects are being attached as service projects to the correct host projects? 
Maybe using https://cloud.google.com/sdk/gcloud/reference/compute/shared-vpc/get-host-project or https://cloud.google.com/sdk/gcloud/reference/compute/shared-vpc/list-associated-resources to assert our logic of attachment and labels are right. WDYT?

Reply:
Makes totally sense @bharathkkb! I'll add it!